domain.go

```diff
@@ -88,10 +88,10 @@ type Config struct {
 	BulkSizeInMB      int    `short:"b" long:"bulk_size" description:"bulk size in MB" default:"5"`
 	ScrollTime        string `short:"t" long:"time"    description:"scroll time" default:"1m"`
 	ScrollSliceSize   int    `long:"sliced_scroll_size"    description:"size of sliced scroll, to make it work, the size should be > 1" default:"1"`
-	RecreateIndex     bool      `short:"f" long:"force"   description:"delete destination index before copying" default:"false"`
+	RecreateIndex     bool      `short:"f" long:"force"   description:"delete destination index before copying" `
```

Comment:
can you keep this default to `false`, same below

Reply:
Hi, the code of packge `go-flags` had been changed,  boolean flag always default to `false' .
https://github.com/jessevdk/go-flags/blob/master/group.go
> 
		if option.isBool() && option.Default != nil {
			return newErrorf(ErrInvalidTag,
				"boolean flag `%s' may not have default values, they always default to `false' and can only be turned on",
				option.shortAndLongName())
		}

Reply:
++,Good!